pkg/apis/knative/v1alpha1/deployer_types.go

```diff
@@ -45,6 +45,8 @@ type DeployerSpec struct {
 	// deployer.
 	Build *Build `json:"build,omitempty"`
 
+	Scale *Scale `json:"scale,omitempty"`
```

Comment:
Since nil has no semantic meaning, we can simplify access by not using a pointer

```suggestion
	Scale Scale `json:"scale,omitempty"`
```